CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpC
 * Raised minimum PHP version to 5.6.
 * **BC break:** Removed the `Interface` suffix from all interfaces.
 * **BC break:** Renamed `HashGenerator` to `DefaultHashGenerator`.
-* Added `HashGenerator` interface.
+
```

Comment:
lets keep this line to make it easier for people who update only when 2.0 stable is out